common/sqliteutil.h

```diff
@@ -0,0 +1,97 @@
+#ifndef ANDEME_COMMON_SQLITE_UTIL_H
+#define ANDEME_COMMON_SQLITE_UTIL_H
+#include <sqlite3.h>
+#include <string>
+#include <vector>
+#include <iostream>
+#include <functional>
+
+#include <schema/message.pb.h>
+
+namespace andeme {
+
+  using Row = std::vector<std::string>;
+  using Callback = std::function<bool(const Row&)>;
+
+  class Sqlite3Statement;
+  class SQLite3Common
```

Comment:
```suggestion
  class SQLite3Storage
```
Давай переименуем в Storage. А то common как-то не ясно, что делает